test/failed-auth.test.js

```diff
@@ -3,7 +3,7 @@
 const test = require('tap').test
 const anger = require('..')
 
-test('failed auth', { timeout: 3000 }, (t) => {
+test('failed auth', { timeout: 20000 }, (t) => {
```

Comment:
This test needs a longer timeout because with auth failing, it will backoff slowly, showing the backoff working 👍